app/test_main.py

```diff
@@ -1,3 +1,31 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (27, 27, [2, 2]),
+        (28, 28, [3, 2])
```

Comment:
what about big/negative numbers ?
![image](https://github.com/mate-academy/py-cat-and-dog-years/assets/103201010/704ca845-9dcb-4102-b9ae-0658056e13b9)